Formula/p/pixi.rb

```diff
@@ -4,6 +4,7 @@ class Pixi < Formula
   url "https://github.com/prefix-dev/pixi/archive/refs/tags/v0.34.0.tar.gz"
   sha256 "2e38380143ab8df1d42609cec3a37704ab5594cc4a247c56937ea9ff063d2688"
   license "BSD-3-Clause"
+  revision 1
```

Comment:
This change should have no user-facing changes, correct?

If so:
```suggestion
```
Because we don't need to make everyone (who already has this installed) reinstall this to pick up a no-op change.

Reply:
true 👍🏻 (except for the HEAD installations but this doesn't ship bottles anyway)